lib/Federation/Proxy/TalkV1/UserConverter.php

```diff
@@ -195,4 +195,11 @@ protected function getLocalParticipants(Room $room): array {
 
 		return $this->participantsPerRoom[$room->getToken()];
 	}
+
+	protected function createCloudIdFromUserIdAndFullServerUrl(string $userId, string $serverUrl): string {
+		if (str_starts_with($serverUrl, 'https://')) {
+			$serverUrl = substr($serverUrl, strlen('https://'));
+		}
+		return $userId . '@' . $serverUrl;
+	}
```

Comment:
So server URL either with HTTPS or without protocol at all, right?

Reply:
in the cloudId https is removed, otherwise it has to be with protocol in all cases